spec.html

```diff
@@ -6211,6 +6222,8 @@
         1. Add _pending_ at the back of the Job Queue named by _queueName_.
         1. Return NormalCompletion(~empty~).
       </emu-alg>
+
+      <emu-note>For host environments consisting of multiple continents, when those host environments enqueue their own jobs, they should generally specify which continent's job queues they are enqueuing on.</emu-note>
```

Comment:
It's not explicitly stated anywhere, that's for sure. It's never stated which continent's job queues are appended to at all, because the spec doesn't currently have a notion of continent, and only deals with a single set of job queues.